libs/features/mas/src/hydrate.js

```diff
@@ -82,32 +82,39 @@ export function processSubtitle(fields, merchCard, subtitleConfig) {
 }
 
 export function processBackgroundImage(
-    fields,
-    merchCard,
-    backgroundImageConfig,
-    variant,
+  fields,
+  merchCard,
+  backgroundImageConfig,
+  variant,
 ) {
-    if (fields.backgroundImage) {
-        switch (variant) {
-            case 'ccd-slice':
-                if (backgroundImageConfig) {
-                    merchCard.append(
-                        createTag(
-                            backgroundImageConfig.tag,
-                            { slot: backgroundImageConfig.slot },
-                            `<img loading="lazy" src="${fields.backgroundImage}" />`,
-                        ),
-                    );
-                }
-                break;
-            case 'ccd-suggested':
-                merchCard.setAttribute(
-                    'background-image',
-                    fields.backgroundImage,
-                );
-                break;
-        }
-    }
+  if (fields.backgroundImage) {
+      switch (variant) {
+          case 'ccd-slice':
+              if (backgroundImageConfig) {
+                  const imgAttributes = {
+                      loading: 'lazy',
+                      src: fields.backgroundImage,
+                  };
+                  if (fields.backgroundImageAltText) {
```

Comment:
jira mentions that if backgroundImageAltText is missing we still need to set `role="none"` attribute
>Ensure <img> elements have alternate text or a role of none or presentation

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

Reply:
RIght, added.